Azure/sonic-buildimage
# [Nokia][IXR7250E] Update Nokia IXR7250e supervisor card device data

#### Why I did it
Updated the Nokia IXR7250E Supervisor card installer.conf file with correct value of the CONSOLE_PORT 

#### How I did it
Modified the  file device/nokia/x86_64-nokia_ixr7250e_sup-r0/installer.conf and the change the CONSOLE_PORT value form 0x3f8 to x2f8

#### How to verify it
Boot up the image with the new change, and make sure that there is no issue to connect the console port.  

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
